docs/notebooks/word2vec.ipynb

```diff
@@ -1295,6 +1295,34 @@
     "print(train_times_table)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Adding Word2Vec \"model to dict\" method to production pipeline"
```

Comment:
Please add more motivation (why this can be useful, it will be good if you show concrete use-case, how you'll use `most_similars_precalc` )